### PR TITLE
Minor docs fixes.

### PR DESCRIPTION
**Pull request by amanda-tarafa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,14 @@ Contributing
 ------------
 
 Contributions to this library are always welcome and highly encouraged.
+See the	[CONTRIBUTING](https://github.com/googleapis/gax-dotnet/blob/master/.github/CONTRIBUTING.md)
```
Comment (jskeet) [inline README.md:26]:
```suggestion
See the [CONTRIBUTING](https://github.com/googleapis/gax-dotnet/blob/master/.github/CONTRIBUTING.md)
```

**Comment by amanda-tarafa** [comment]
Requested review from @JustinBeckwith .

Justin, main things for you to confirm here are:

- Why is the CONTRIBUTING file in .github instead of in the repo root. (According to [this](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors) it makes no functional difference, so I suppose it's in .github because it's there in all the other repos.)
- Why dindn't the CONTRIBUTING file have an md extension (I believe this to be just an oversight and it means that the file is being rendered as text and not as markdown).

**Comment by JustinBeckwith** [comment]
1. I tend to prefer putting the CONTRIBUTING.md in the root of the repository because it's more obvious for people who visit.  Also, there are tools in OSPO that assume the root as the location.  

2. Yeah I'd add the `.md` :) for better or for worse, we write a single CONTRIBUTING.md, and share it across every node.js repository.  It's worth considering how you keep these up to date broadly :)

**Comment by amanda-tarafa** [comment]
1. @JustinBeckwith It seems it was you who moved CONTRIBUTING from root to .github in #280 :)

**Comment by JustinBeckwith** [comment]
Lol I'm sure I had a reason nearly 2 years ago.  Who knows what it was 😆  Remember kids, write better commit messages.

**Pull request merged**